nfts/src/tests/extrinsics.rs

```diff
@@ -9,375 +9,665 @@ fn origin(account: u64) -> mock::Origin {
 	RawOrigin::Signed(account).into()
 }
 
-/* fn root() -> mock::Origin {
+fn root() -> mock::Origin {
 	RawOrigin::Root.into()
-} */
+}
 
-mod transfer {
+mod create {
 	use super::*;
 
 	#[test]
-	fn cannot_transfer_delegated_nfts() {
-		ExtBuilder::new_build(vec![(ALICE, 100)]).execute_with(|| {
-			let nft_id = NFTs::create_nft(ALICE, vec![0], None).unwrap();
-			assert_ok!(NFTs::set_viewer(nft_id, Some(BOB)));
+	fn create_ok_with_no_series() {
+		ExtBuilder::default().caps(vec![(ALICE, 1000)]).build().execute_with(|| {
+			// State checks
+			assert_eq!(NFTs::nft_id_generator(), 0);
+			assert_eq!(NFTs::series_id_generator(), 0);
```

Comment:
You don't need that.

Reply:
Done, it was already there before rework, I did not know why so I let it, now removed.